components/utils/lazysections.tsx

```diff
@@ -44,12 +50,9 @@ const Lazysections = ({ children }) => {
       })
     }
   }, [onScreen, controls])
+
   return (
     <motion.div
-      threshold="0.5"
-      delay="100"
-      rootMargin="0px"
-      triggerOnce="true"
```

Comment:
Mentioned in [discord](https://discord.com/channels/808364903822917662/808364903822917665/899234527837315103), these don't appear to be valid props for `motion.div`.  Happy to change if someone can tell me how they are supposed to be applied.

Reply:
As long as everything works still, should be fine to remove